.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,87 @@
+# Ultralytics 🚀 AGPL-3.0 License - https://ultralytics.com/license
+
+name: 🐛 Bug Report
+description: "Problems with the Ultralytics Handbook site"
+labels: [bug, triage]
+type: "bug"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting an Ultralytics Handbook 🐛 Bug Report!
+
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      description: >
+        Please search the Ultralytics Handbook [site](https://handbook.ultralytics.com/) and [issues](https://github.com/ultralytics/handbook/issues) to see if a similar bug report already exists.
+      options:
+        - label: >
+            I have searched https://github.com/ultralytics/handbook/issues and did not find a similar report.
+          required: true
+
+  - type: dropdown
+    attributes:
+      label: Project area
+      description: |
+        Help us route the report to the right maintainers.
+      multiple: true
+      options:
+        - "Content pages"
+        - "Navigation or search"
+        - "Onboarding/people guides"
+        - "Build tooling (Zensical)"
+        - "Deployment (Vercel)"
+        - "Other"
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Bug
+      description: Please describe the issue in detail so we can reproduce it in Ultralytics Handbook. Include logs, screenshots, console output, and any context that helps explain the problem.
+      placeholder: |
+        💡 ProTip! Include as much information as possible (logs, tracebacks, screenshots, etc.) to receive the most helpful response.
+    validations:
+      required: true
+
+  - type: textarea
```

💭 **SUGGESTION**: The `Environment` section is required (`required: true`), which might be unnecessarily strict for some purely content-related Handbook issues (e.g., typos, broken links). Consider making it optional to reduce friction for simple reports.

**Suggested change:**
```suggestion
  - type: textarea
```